lib/orogen/templates/proxies/Task.cpp

```diff
@@ -0,0 +1,129 @@
+#include "<%= task.basename %>.hpp"
+
+
+#include <rtt/typekit/RealTimeTypekit.hpp>
+<% if component.enabled_transports.include?('corba') %>
+#include <rtt/transports/corba/TransportPlugin.hpp>
+<% end %>
+<% if component.enabled_transports.include?('mqueue') %>
+#include <rtt/transports/mqueue/TransportPlugin.hpp>
+<% end %>
+
+<% if component.typekit || !component.used_typekits.empty? %>
+#include <rtt/types/TypekitPlugin.hpp>
+<% end %>
+<% if typekit = component.typekit %>
+#include <<%= typekit.name %>/typekit/Plugin.hpp>
+<% component.enabled_transports.each do |transport_name| %>
+#include "typekit/transports/<%= transport_name %>/TransportPlugin.hpp"
+<% end %>
+<% end %>
+<% component.used_typekits.each do |tk| %>
+    <% next if tk.virtual? %>
+    <% next if tk.name == "logger" %>
+#include <<%= tk.name %>/typekit/Plugin.hpp>
+    <% component.enabled_transports.each do |transport_name| %>
+#include <<%= tk.name %>/transports/<%= transport_name %>/TransportPlugin.hpp>
+    <% end %>
+<% end %>
+
+namespace <%= component.name %> {
+
+namespace proxies {
+
+<%= task.basename %>Initializer::<%= task.basename %>Initializer(std::string location, bool is_ior)
+:TaskContextProxy()
+{
+    initTypes();
+    
+    initFromURIOrTaskname(location, is_ior);
+}
+
+void <%= task.basename %>Initializer::initTypes()
+{
+    RTT::types::TypekitRepository::Import( new RTT::types::RealTimeTypekitPlugin );
```

Comment:
nope, import has a check included and deletes the typekit / transport if is already registered